MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1069 @@
+    dataForm.ItemsSourceProvider = new SourceProviderExt();
+    dataForm.RegisterEditor("Country", DataFormEditorType.AutoComplete);
+    dataForm.GenerateDataFormItem += DataForm_GenerateDataFormItem;
+
+    private void DataForm_GenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Country")
+        {
+            var autoCompleteItem = (e.DataFormItem as DataFormAutoCompleteItem);
+            autoCompleteItem.AutoCompleteMode = AutoCompleteMode.Suggest;
+        }
+    }
+    public class SourceProviderExt : IDataFormSourceProvider
+    {
+         public object GetSource(string sourceName)
+        {
+            var list = new List<string>();
+            if (sourceName == "Country")
+            {
+                list.Add("Indonesia");
+                list.Add("Italy");
+                list.Add("India");
+                list.Add("Iran");
+                list.Add("Iraq");
+                list.Add("Uganda");
+                list.Add("Ukraine");
+                list.Add("Canada");
+                list.Add("Australia");
+                list.Add("Uzbekistan");
+                list.Add("France");
+                list.Add("United Kingdom");
+                list.Add("United States");
+
+            }
+            return list;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+#### AutoComplete editor suggestion options
+
+The phenomenon of string comparison for filtering suggestions can be changed using the "StartsWith" and "Containswith" `Link` property. The default filtering strategy is “StartsWith”, and it is case insensitive. The available filtering modes are,
+
+* StartsWith
+
+* ContainsWithCaseSensitive
+
+#### Filtering words that starts with the input text
+
+Displays all the matches that contain first character of the typed characters in items source of autocomplete editor. This strategy is case in-sensitive.
+
+#### Filtering words that contains with the input text
+
+Displays all the matches that contain the typed characters in items source of autocomplete editor. This strategy is case in-sensitive.
+	
+## Custom editor
+
+The custom editor can be added to DataForm by inheriting the `IDataFormEditor` class for business models. You can create custom editor using implement all method in IDataFormEditor. 
+
+To add custom editor in DataForm, register the editor with custom registered type for the required property using `RegisterEditor` method. You can also customize editor settings by using available methods in `IDataFormEditor`. 
+
+### Creating custom editor using IDataFormEditor 
+
+Views such as labels, buttons, and sliders can be loaded to custom editor. Here, entry is loaded as custom editor for `PhoneNumer` property. 
+
+### Below methioned example code for Numeric editor which accepts only numbers 
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+    public class NumericTextEditor : IDataFormEditor 
```

Comment:
replace this entire class with Sb numeric editor

Reply:
Sb numeric editor added